[2017-07-17 14:02:29] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /lucky/number"" at /var/www/html/app/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/EventListener/RouterListener.php line 125 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /lucky/number\" at /var/www/html/app/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/EventListener/RouterListener.php:125, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /var/www/html/app/var/cache/prod/appProdProjectContainerUrlMatcher.php:47)"} []
[2017-07-17 14:04:56] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /lucky/number"" at /var/www/html/app/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/EventListener/RouterListener.php line 125 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /lucky/number\" at /var/www/html/app/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/EventListener/RouterListener.php:125, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /var/www/html/app/var/cache/prod/appProdProjectContainerUrlMatcher.php:47)"} []
[2017-08-04 11:42:10] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /contactar/a"" at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/EventListener/RouterListener.php line 125 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /contactar/a\" at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/EventListener/RouterListener.php:125, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /var/www/html/app/app/var/cache/prod/appProdProjectContainerUrlMatcher.php:47)"} []
[2017-08-04 11:49:37] request.INFO: Matched route "contactar". {"route":"contactar","route_parameters":{"_controller":"AppBundle\\Controller\\EmpresaController::contactarAction","marca":"a","_route":"contactar"},"request_uri":"http://127.0.0.1:8000/app.php/contactar/a","method":"GET"} []
[2017-08-04 11:49:37] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-08-04 11:49:37] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "Te has equivocado. No existe" at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php line 300 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): Te has equivocado. No existe at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php:300)"} []
[2017-08-04 11:49:40] request.INFO: Matched route "contactar". {"route":"contactar","route_parameters":{"_controller":"AppBundle\\Controller\\EmpresaController::contactarAction","marca":"a","_route":"contactar"},"request_uri":"http://127.0.0.1:8000/app.php/contactar/a","method":"GET"} []
[2017-08-04 11:49:40] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-08-04 11:49:40] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "Te has equivocado. No existe" at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php line 300 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): Te has equivocado. No existe at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php:300)"} []
[2017-08-04 11:52:20] request.INFO: Matched route "contactar". {"route":"contactar","route_parameters":{"_controller":"AppBundle\\Controller\\EmpresaController::contactarAction","marca":"a","_route":"contactar"},"request_uri":"http://127.0.0.1:8000/app.php/contactar/a","method":"GET"} []
[2017-08-04 11:52:20] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-08-04 11:52:20] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "Te has equivocado. No existe" at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php line 300 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): Te has equivocado. No existe at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php:300)"} []
[2017-08-04 11:52:23] request.INFO: Matched route "contactar". {"route":"contactar","route_parameters":{"_controller":"AppBundle\\Controller\\EmpresaController::contactarAction","marca":"a","_route":"contactar"},"request_uri":"http://127.0.0.1:8000/app.php/contactar/a","method":"GET"} []
[2017-08-04 11:52:23] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-08-04 11:52:23] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "Te has equivocado. No existe" at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php line 300 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): Te has equivocado. No existe at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php:300)"} []
[2017-08-04 11:52:43] request.INFO: Matched route "contactar". {"route":"contactar","route_parameters":{"_controller":"AppBundle\\Controller\\EmpresaController::contactarAction","marca":"a","_route":"contactar"},"request_uri":"http://127.0.0.1:8000/app.php/contactar/a","method":"GET"} []
[2017-08-04 11:52:43] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-08-04 11:52:43] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "Te has equivocado. No existe" at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php line 300 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): Te has equivocado. No existe at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php:300)"} []
[2017-08-04 11:52:44] request.INFO: Matched route "contactar". {"route":"contactar","route_parameters":{"_controller":"AppBundle\\Controller\\EmpresaController::contactarAction","marca":"a","_route":"contactar"},"request_uri":"http://127.0.0.1:8000/app.php/contactar/a","method":"GET"} []
[2017-08-04 11:52:44] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-08-04 11:52:44] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "Te has equivocado. No existe" at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php line 300 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): Te has equivocado. No existe at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php:300)"} []
[2017-08-04 11:52:44] request.INFO: Matched route "contactar". {"route":"contactar","route_parameters":{"_controller":"AppBundle\\Controller\\EmpresaController::contactarAction","marca":"a","_route":"contactar"},"request_uri":"http://127.0.0.1:8000/app.php/contactar/a","method":"GET"} []
[2017-08-04 11:52:44] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-08-04 11:52:44] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "Te has equivocado. No existe" at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php line 300 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): Te has equivocado. No existe at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php:300)"} []
[2017-08-04 11:52:44] request.INFO: Matched route "contactar". {"route":"contactar","route_parameters":{"_controller":"AppBundle\\Controller\\EmpresaController::contactarAction","marca":"a","_route":"contactar"},"request_uri":"http://127.0.0.1:8000/app.php/contactar/a","method":"GET"} []
[2017-08-04 11:52:44] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-08-04 11:52:44] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "Te has equivocado. No existe" at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php line 300 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): Te has equivocado. No existe at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php:300)"} []
[2017-08-04 11:53:40] request.INFO: Matched route "contactar". {"route":"contactar","route_parameters":{"_controller":"AppBundle\\Controller\\EmpresaController::contactarAction","marca":"a","_route":"contactar"},"request_uri":"http://127.0.0.1:8000/app.php/contactar/a","method":"GET"} []
[2017-08-04 11:53:41] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-08-04 11:53:41] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "Te has equivocado. No existe" at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php line 300 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): Te has equivocado. No existe at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php:300)"} []
[2017-08-04 11:57:25] request.INFO: Matched route "contactar". {"route":"contactar","route_parameters":{"_controller":"AppBundle\\Controller\\EmpresaController::contactarAction","marca":"a","_route":"contactar"},"request_uri":"http://127.0.0.1:8000/app.php/contactar/a","method":"GET"} []
[2017-08-04 11:57:25] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-08-04 11:57:25] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "Te has equivocado. No existe" at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php line 300 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): Te has equivocado. No existe at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php:300)"} []
[2017-08-04 11:57:25] request.CRITICAL: Exception thrown when handling an exception (Twig_Error_Loader: The "/var/www/html/app/app/app/Resources/views" directory does not exist ("/var/www/html/app/app/app/Resources/views"). at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php line 101) {"exception":"[object] (Twig_Error_Loader(code: 0): The \"/var/www/html/app/app/app/Resources/views\" directory does not exist (\"/var/www/html/app/app/app/Resources/views\"). at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php:101)"} []
[2017-08-04 11:57:25] php.CRITICAL: Uncaught Exception: The "/var/www/html/app/app/app/Resources/views" directory does not exist ("/var/www/html/app/app/app/Resources/views"). {"exception":"[object] (Twig_Error_Loader(code: 0): The \"/var/www/html/app/app/app/Resources/views\" directory does not exist (\"/var/www/html/app/app/app/Resources/views\"). at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php:101, Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): Te has equivocado. No existe at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php:300)"} []
[2017-08-04 11:57:25] request.CRITICAL: Uncaught PHP Exception Twig_Error_Loader: "The "/var/www/html/app/app/app/Resources/views" directory does not exist ("/var/www/html/app/app/app/Resources/views")." at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php line 101 {"exception":"[object] (Twig_Error_Loader(code: 0): The \"/var/www/html/app/app/app/Resources/views\" directory does not exist (\"/var/www/html/app/app/app/Resources/views\"). at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php:101, Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): Te has equivocado. No existe at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php:300)"} []
[2017-08-04 11:57:25] request.CRITICAL: Exception thrown when handling an exception (Twig_Error_Loader: The "/var/www/html/app/app/app/Resources/views" directory does not exist ("/var/www/html/app/app/app/Resources/views"). at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php line 101) {"exception":"[object] (Twig_Error_Loader(code: 0): The \"/var/www/html/app/app/app/Resources/views\" directory does not exist (\"/var/www/html/app/app/app/Resources/views\"). at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php:101)"} []
[2017-08-04 11:57:25] php.CRITICAL: Uncaught Exception: The "/var/www/html/app/app/app/Resources/views" directory does not exist ("/var/www/html/app/app/app/Resources/views"). {"exception":"[object] (Twig_Error_Loader(code: 0): The \"/var/www/html/app/app/app/Resources/views\" directory does not exist (\"/var/www/html/app/app/app/Resources/views\"). at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php:101, Twig_Error_Loader(code: 0): The \"/var/www/html/app/app/app/Resources/views\" directory does not exist (\"/var/www/html/app/app/app/Resources/views\"). at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php:101, Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): Te has equivocado. No existe at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php:300)"} []
[2017-08-04 11:57:25] php.CRITICAL: Fatal Error: Uncaught exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' with message 'Te has equivocado. No existe' in /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php:300 Stack trace: #0 /var/www/html/app/app/src/AppBundle/Controller/EmpresaController.php(66): Symfony\Bundle\FrameworkBundle\Controller\Controller->createNotFoundException('Te has equivoca...') #1 [internal function]: AppBundle\Controller\EmpresaController->contactarAction('a') #2 /var/www/html/app/app/var/cache/prod/classes.php(4916): call_user_func_array(Array, Array) #3 /var/www/html/app/app/var/cache/prod/classes.php(4871): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1) #4 /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php(171): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #5 /var/www/html/app/app/web/app.php(19): S {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Error: Uncaught exception 'Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException' with message 'Te has equivocado. No existe' in /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php:300\nStack trace:\n#0 /var/www/html/app/app/src/AppBundle/Controller/EmpresaController.php(66): Symfony\\Bundle\\FrameworkBundle\\Controller\\Controller->createNotFoundException('Te has equivoca...')\n#1 [internal function]: AppBundle\\Controller\\EmpresaController->contactarAction('a')\n#2 /var/www/html/app/app/var/cache/prod/classes.php(4916): call_user_func_array(Array, Array)\n#3 /var/www/html/app/app/var/cache/prod/classes.php(4871): Symfony\\Component\\HttpKernel\\HttpKernel->handleRaw(Object(Symfony\\Component\\HttpFoundation\\Request), 1)\n#4 /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php(171): Symfony\\Component\\HttpKernel\\HttpKernel->handle(Object(Symfony\\Component\\HttpFoundation\\Request), 1, true)\n#5 /var/www/html/app/app/web/app.php(19): S at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php:101)"} []
[2017-08-04 11:57:28] request.INFO: Matched route "contactar". {"route":"contactar","route_parameters":{"_controller":"AppBundle\\Controller\\EmpresaController::contactarAction","marca":"a","_route":"contactar"},"request_uri":"http://127.0.0.1:8000/app.php/contactar/a","method":"GET"} []
[2017-08-04 11:57:28] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-08-04 11:57:28] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "Te has equivocado. No existe" at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php line 300 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): Te has equivocado. No existe at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php:300)"} []
[2017-08-04 11:57:28] request.CRITICAL: Exception thrown when handling an exception (Twig_Error_Loader: The "/var/www/html/app/app/app/Resources/views" directory does not exist ("/var/www/html/app/app/app/Resources/views"). at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php line 101) {"exception":"[object] (Twig_Error_Loader(code: 0): The \"/var/www/html/app/app/app/Resources/views\" directory does not exist (\"/var/www/html/app/app/app/Resources/views\"). at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php:101)"} []
[2017-08-04 11:57:28] php.CRITICAL: Uncaught Exception: The "/var/www/html/app/app/app/Resources/views" directory does not exist ("/var/www/html/app/app/app/Resources/views"). {"exception":"[object] (Twig_Error_Loader(code: 0): The \"/var/www/html/app/app/app/Resources/views\" directory does not exist (\"/var/www/html/app/app/app/Resources/views\"). at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php:101, Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): Te has equivocado. No existe at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php:300)"} []
[2017-08-04 11:57:28] request.CRITICAL: Uncaught PHP Exception Twig_Error_Loader: "The "/var/www/html/app/app/app/Resources/views" directory does not exist ("/var/www/html/app/app/app/Resources/views")." at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php line 101 {"exception":"[object] (Twig_Error_Loader(code: 0): The \"/var/www/html/app/app/app/Resources/views\" directory does not exist (\"/var/www/html/app/app/app/Resources/views\"). at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php:101, Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): Te has equivocado. No existe at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php:300)"} []
[2017-08-04 11:57:28] request.CRITICAL: Exception thrown when handling an exception (Twig_Error_Loader: The "/var/www/html/app/app/app/Resources/views" directory does not exist ("/var/www/html/app/app/app/Resources/views"). at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php line 101) {"exception":"[object] (Twig_Error_Loader(code: 0): The \"/var/www/html/app/app/app/Resources/views\" directory does not exist (\"/var/www/html/app/app/app/Resources/views\"). at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php:101)"} []
[2017-08-04 11:57:28] php.CRITICAL: Uncaught Exception: The "/var/www/html/app/app/app/Resources/views" directory does not exist ("/var/www/html/app/app/app/Resources/views"). {"exception":"[object] (Twig_Error_Loader(code: 0): The \"/var/www/html/app/app/app/Resources/views\" directory does not exist (\"/var/www/html/app/app/app/Resources/views\"). at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php:101, Twig_Error_Loader(code: 0): The \"/var/www/html/app/app/app/Resources/views\" directory does not exist (\"/var/www/html/app/app/app/Resources/views\"). at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php:101, Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): Te has equivocado. No existe at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php:300)"} []
[2017-08-04 11:57:28] php.CRITICAL: Fatal Error: Uncaught exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' with message 'Te has equivocado. No existe' in /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php:300 Stack trace: #0 /var/www/html/app/app/src/AppBundle/Controller/EmpresaController.php(66): Symfony\Bundle\FrameworkBundle\Controller\Controller->createNotFoundException('Te has equivoca...') #1 [internal function]: AppBundle\Controller\EmpresaController->contactarAction('a') #2 /var/www/html/app/app/var/cache/prod/classes.php(4916): call_user_func_array(Array, Array) #3 /var/www/html/app/app/var/cache/prod/classes.php(4871): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1) #4 /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php(171): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #5 /var/www/html/app/app/web/app.php(19): S {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Error: Uncaught exception 'Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException' with message 'Te has equivocado. No existe' in /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php:300\nStack trace:\n#0 /var/www/html/app/app/src/AppBundle/Controller/EmpresaController.php(66): Symfony\\Bundle\\FrameworkBundle\\Controller\\Controller->createNotFoundException('Te has equivoca...')\n#1 [internal function]: AppBundle\\Controller\\EmpresaController->contactarAction('a')\n#2 /var/www/html/app/app/var/cache/prod/classes.php(4916): call_user_func_array(Array, Array)\n#3 /var/www/html/app/app/var/cache/prod/classes.php(4871): Symfony\\Component\\HttpKernel\\HttpKernel->handleRaw(Object(Symfony\\Component\\HttpFoundation\\Request), 1)\n#4 /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php(171): Symfony\\Component\\HttpKernel\\HttpKernel->handle(Object(Symfony\\Component\\HttpFoundation\\Request), 1, true)\n#5 /var/www/html/app/app/web/app.php(19): S at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php:101)"} []
[2017-08-04 11:57:46] request.INFO: Matched route "contactar". {"route":"contactar","route_parameters":{"_controller":"AppBundle\\Controller\\EmpresaController::contactarAction","marca":"a","_route":"contactar"},"request_uri":"http://127.0.0.1:8000/app.php/contactar/a","method":"GET"} []
[2017-08-04 11:57:46] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-08-04 11:57:46] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "Te has equivocado. No existe" at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php line 300 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): Te has equivocado. No existe at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php:300)"} []
[2017-08-04 11:57:46] request.CRITICAL: Exception thrown when handling an exception (Twig_Error_Loader: The "/var/www/html/app/app/app/Resources/views" directory does not exist ("/var/www/html/app/app/app/Resources/views"). at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php line 101) {"exception":"[object] (Twig_Error_Loader(code: 0): The \"/var/www/html/app/app/app/Resources/views\" directory does not exist (\"/var/www/html/app/app/app/Resources/views\"). at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php:101)"} []
[2017-08-04 11:57:46] php.CRITICAL: Uncaught Exception: The "/var/www/html/app/app/app/Resources/views" directory does not exist ("/var/www/html/app/app/app/Resources/views"). {"exception":"[object] (Twig_Error_Loader(code: 0): The \"/var/www/html/app/app/app/Resources/views\" directory does not exist (\"/var/www/html/app/app/app/Resources/views\"). at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php:101, Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): Te has equivocado. No existe at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php:300)"} []
[2017-08-04 11:57:46] request.CRITICAL: Uncaught PHP Exception Twig_Error_Loader: "The "/var/www/html/app/app/app/Resources/views" directory does not exist ("/var/www/html/app/app/app/Resources/views")." at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php line 101 {"exception":"[object] (Twig_Error_Loader(code: 0): The \"/var/www/html/app/app/app/Resources/views\" directory does not exist (\"/var/www/html/app/app/app/Resources/views\"). at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php:101, Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): Te has equivocado. No existe at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php:300)"} []
[2017-08-04 11:57:46] request.CRITICAL: Exception thrown when handling an exception (Twig_Error_Loader: The "/var/www/html/app/app/app/Resources/views" directory does not exist ("/var/www/html/app/app/app/Resources/views"). at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php line 101) {"exception":"[object] (Twig_Error_Loader(code: 0): The \"/var/www/html/app/app/app/Resources/views\" directory does not exist (\"/var/www/html/app/app/app/Resources/views\"). at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php:101)"} []
[2017-08-04 11:57:46] php.CRITICAL: Uncaught Exception: The "/var/www/html/app/app/app/Resources/views" directory does not exist ("/var/www/html/app/app/app/Resources/views"). {"exception":"[object] (Twig_Error_Loader(code: 0): The \"/var/www/html/app/app/app/Resources/views\" directory does not exist (\"/var/www/html/app/app/app/Resources/views\"). at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php:101, Twig_Error_Loader(code: 0): The \"/var/www/html/app/app/app/Resources/views\" directory does not exist (\"/var/www/html/app/app/app/Resources/views\"). at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php:101, Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): Te has equivocado. No existe at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php:300)"} []
[2017-08-04 11:57:46] php.CRITICAL: Fatal Error: Uncaught exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' with message 'Te has equivocado. No existe' in /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php:300 Stack trace: #0 /var/www/html/app/app/src/AppBundle/Controller/EmpresaController.php(66): Symfony\Bundle\FrameworkBundle\Controller\Controller->createNotFoundException('Te has equivoca...') #1 [internal function]: AppBundle\Controller\EmpresaController->contactarAction('a') #2 /var/www/html/app/app/var/cache/prod/classes.php(4916): call_user_func_array(Array, Array) #3 /var/www/html/app/app/var/cache/prod/classes.php(4871): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1) #4 /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php(171): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #5 /var/www/html/app/app/web/app.php(19): S {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Error: Uncaught exception 'Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException' with message 'Te has equivocado. No existe' in /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php:300\nStack trace:\n#0 /var/www/html/app/app/src/AppBundle/Controller/EmpresaController.php(66): Symfony\\Bundle\\FrameworkBundle\\Controller\\Controller->createNotFoundException('Te has equivoca...')\n#1 [internal function]: AppBundle\\Controller\\EmpresaController->contactarAction('a')\n#2 /var/www/html/app/app/var/cache/prod/classes.php(4916): call_user_func_array(Array, Array)\n#3 /var/www/html/app/app/var/cache/prod/classes.php(4871): Symfony\\Component\\HttpKernel\\HttpKernel->handleRaw(Object(Symfony\\Component\\HttpFoundation\\Request), 1)\n#4 /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php(171): Symfony\\Component\\HttpKernel\\HttpKernel->handle(Object(Symfony\\Component\\HttpFoundation\\Request), 1, true)\n#5 /var/www/html/app/app/web/app.php(19): S at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php:101)"} []
[2017-08-04 11:58:25] request.INFO: Matched route "contactar". {"route":"contactar","route_parameters":{"_controller":"AppBundle\\Controller\\EmpresaController::contactarAction","marca":"a","_route":"contactar"},"request_uri":"http://127.0.0.1:8000/app.php/contactar/a","method":"GET"} []
[2017-08-04 11:58:25] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-08-04 11:58:25] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "Te has equivocado. No existe" at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php line 300 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): Te has equivocado. No existe at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php:300)"} []
[2017-08-04 11:58:25] request.CRITICAL: Exception thrown when handling an exception (Twig_Error_Loader: The "/var/www/html/app/app/app/Resources/views" directory does not exist ("/var/www/html/app/app/app/Resources/views"). at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php line 101) {"exception":"[object] (Twig_Error_Loader(code: 0): The \"/var/www/html/app/app/app/Resources/views\" directory does not exist (\"/var/www/html/app/app/app/Resources/views\"). at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php:101)"} []
[2017-08-04 11:58:25] php.CRITICAL: Uncaught Exception: The "/var/www/html/app/app/app/Resources/views" directory does not exist ("/var/www/html/app/app/app/Resources/views"). {"exception":"[object] (Twig_Error_Loader(code: 0): The \"/var/www/html/app/app/app/Resources/views\" directory does not exist (\"/var/www/html/app/app/app/Resources/views\"). at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php:101, Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): Te has equivocado. No existe at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php:300)"} []
[2017-08-04 11:58:25] request.CRITICAL: Uncaught PHP Exception Twig_Error_Loader: "The "/var/www/html/app/app/app/Resources/views" directory does not exist ("/var/www/html/app/app/app/Resources/views")." at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php line 101 {"exception":"[object] (Twig_Error_Loader(code: 0): The \"/var/www/html/app/app/app/Resources/views\" directory does not exist (\"/var/www/html/app/app/app/Resources/views\"). at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php:101, Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): Te has equivocado. No existe at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php:300)"} []
[2017-08-04 11:58:25] request.CRITICAL: Exception thrown when handling an exception (Twig_Error_Loader: The "/var/www/html/app/app/app/Resources/views" directory does not exist ("/var/www/html/app/app/app/Resources/views"). at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php line 101) {"exception":"[object] (Twig_Error_Loader(code: 0): The \"/var/www/html/app/app/app/Resources/views\" directory does not exist (\"/var/www/html/app/app/app/Resources/views\"). at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php:101)"} []
[2017-08-04 11:58:25] php.CRITICAL: Uncaught Exception: The "/var/www/html/app/app/app/Resources/views" directory does not exist ("/var/www/html/app/app/app/Resources/views"). {"exception":"[object] (Twig_Error_Loader(code: 0): The \"/var/www/html/app/app/app/Resources/views\" directory does not exist (\"/var/www/html/app/app/app/Resources/views\"). at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php:101, Twig_Error_Loader(code: 0): The \"/var/www/html/app/app/app/Resources/views\" directory does not exist (\"/var/www/html/app/app/app/Resources/views\"). at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php:101, Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): Te has equivocado. No existe at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php:300)"} []
[2017-08-04 11:58:25] php.CRITICAL: Fatal Error: Uncaught exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' with message 'Te has equivocado. No existe' in /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php:300 Stack trace: #0 /var/www/html/app/app/src/AppBundle/Controller/EmpresaController.php(66): Symfony\Bundle\FrameworkBundle\Controller\Controller->createNotFoundException('Te has equivoca...') #1 [internal function]: AppBundle\Controller\EmpresaController->contactarAction('a') #2 /var/www/html/app/app/var/cache/prod/classes.php(4916): call_user_func_array(Array, Array) #3 /var/www/html/app/app/var/cache/prod/classes.php(4871): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1) #4 /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php(171): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #5 /var/www/html/app/app/web/app.php(19): S {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Error: Uncaught exception 'Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException' with message 'Te has equivocado. No existe' in /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php:300\nStack trace:\n#0 /var/www/html/app/app/src/AppBundle/Controller/EmpresaController.php(66): Symfony\\Bundle\\FrameworkBundle\\Controller\\Controller->createNotFoundException('Te has equivoca...')\n#1 [internal function]: AppBundle\\Controller\\EmpresaController->contactarAction('a')\n#2 /var/www/html/app/app/var/cache/prod/classes.php(4916): call_user_func_array(Array, Array)\n#3 /var/www/html/app/app/var/cache/prod/classes.php(4871): Symfony\\Component\\HttpKernel\\HttpKernel->handleRaw(Object(Symfony\\Component\\HttpFoundation\\Request), 1)\n#4 /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php(171): Symfony\\Component\\HttpKernel\\HttpKernel->handle(Object(Symfony\\Component\\HttpFoundation\\Request), 1, true)\n#5 /var/www/html/app/app/web/app.php(19): S at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php:101)"} []
[2017-08-04 11:58:42] request.INFO: Matched route "contactar". {"route":"contactar","route_parameters":{"_controller":"AppBundle\\Controller\\EmpresaController::contactarAction","marca":"a","_route":"contactar"},"request_uri":"http://127.0.0.1:8000/app.php/contactar/a","method":"GET"} []
[2017-08-04 11:58:42] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-08-04 11:58:42] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "Te has equivocado. No existe" at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php line 300 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): Te has equivocado. No existe at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php:300)"} []
[2017-08-04 11:58:42] request.CRITICAL: Exception thrown when handling an exception (Twig_Error_Loader: The "/var/www/html/app/app/app/Resources/views" directory does not exist ("/var/www/html/app/app/app/Resources/views"). at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php line 101) {"exception":"[object] (Twig_Error_Loader(code: 0): The \"/var/www/html/app/app/app/Resources/views\" directory does not exist (\"/var/www/html/app/app/app/Resources/views\"). at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php:101)"} []
[2017-08-04 11:58:42] php.CRITICAL: Uncaught Exception: The "/var/www/html/app/app/app/Resources/views" directory does not exist ("/var/www/html/app/app/app/Resources/views"). {"exception":"[object] (Twig_Error_Loader(code: 0): The \"/var/www/html/app/app/app/Resources/views\" directory does not exist (\"/var/www/html/app/app/app/Resources/views\"). at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php:101, Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): Te has equivocado. No existe at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php:300)"} []
[2017-08-04 11:58:42] request.CRITICAL: Uncaught PHP Exception Twig_Error_Loader: "The "/var/www/html/app/app/app/Resources/views" directory does not exist ("/var/www/html/app/app/app/Resources/views")." at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php line 101 {"exception":"[object] (Twig_Error_Loader(code: 0): The \"/var/www/html/app/app/app/Resources/views\" directory does not exist (\"/var/www/html/app/app/app/Resources/views\"). at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php:101, Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): Te has equivocado. No existe at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php:300)"} []
[2017-08-04 11:58:42] request.CRITICAL: Exception thrown when handling an exception (Twig_Error_Loader: The "/var/www/html/app/app/app/Resources/views" directory does not exist ("/var/www/html/app/app/app/Resources/views"). at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php line 101) {"exception":"[object] (Twig_Error_Loader(code: 0): The \"/var/www/html/app/app/app/Resources/views\" directory does not exist (\"/var/www/html/app/app/app/Resources/views\"). at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php:101)"} []
[2017-08-04 11:58:42] php.CRITICAL: Uncaught Exception: The "/var/www/html/app/app/app/Resources/views" directory does not exist ("/var/www/html/app/app/app/Resources/views"). {"exception":"[object] (Twig_Error_Loader(code: 0): The \"/var/www/html/app/app/app/Resources/views\" directory does not exist (\"/var/www/html/app/app/app/Resources/views\"). at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php:101, Twig_Error_Loader(code: 0): The \"/var/www/html/app/app/app/Resources/views\" directory does not exist (\"/var/www/html/app/app/app/Resources/views\"). at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php:101, Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): Te has equivocado. No existe at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php:300)"} []
[2017-08-04 11:58:42] php.CRITICAL: Fatal Error: Uncaught exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' with message 'Te has equivocado. No existe' in /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php:300 Stack trace: #0 /var/www/html/app/app/src/AppBundle/Controller/EmpresaController.php(66): Symfony\Bundle\FrameworkBundle\Controller\Controller->createNotFoundException('Te has equivoca...') #1 [internal function]: AppBundle\Controller\EmpresaController->contactarAction('a') #2 /var/www/html/app/app/var/cache/prod/classes.php(4916): call_user_func_array(Array, Array) #3 /var/www/html/app/app/var/cache/prod/classes.php(4871): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1) #4 /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php(171): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #5 /var/www/html/app/app/web/app.php(19): S {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Error: Uncaught exception 'Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException' with message 'Te has equivocado. No existe' in /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php:300\nStack trace:\n#0 /var/www/html/app/app/src/AppBundle/Controller/EmpresaController.php(66): Symfony\\Bundle\\FrameworkBundle\\Controller\\Controller->createNotFoundException('Te has equivoca...')\n#1 [internal function]: AppBundle\\Controller\\EmpresaController->contactarAction('a')\n#2 /var/www/html/app/app/var/cache/prod/classes.php(4916): call_user_func_array(Array, Array)\n#3 /var/www/html/app/app/var/cache/prod/classes.php(4871): Symfony\\Component\\HttpKernel\\HttpKernel->handleRaw(Object(Symfony\\Component\\HttpFoundation\\Request), 1)\n#4 /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php(171): Symfony\\Component\\HttpKernel\\HttpKernel->handle(Object(Symfony\\Component\\HttpFoundation\\Request), 1, true)\n#5 /var/www/html/app/app/web/app.php(19): S at /var/www/html/app/app/vendor/twig/twig/lib/Twig/Loader/Filesystem.php:101)"} []
[2017-08-04 11:59:43] request.INFO: Matched route "contactar". {"route":"contactar","route_parameters":{"_controller":"AppBundle\\Controller\\EmpresaController::contactarAction","marca":"a","_route":"contactar"},"request_uri":"http://127.0.0.1:8000/app.php/contactar/a","method":"GET"} []
[2017-08-04 11:59:43] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-08-04 11:59:43] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "Te has equivocado. No existe" at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php line 300 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): Te has equivocado. No existe at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php:300)"} []
[2017-08-04 12:01:00] request.INFO: Matched route "contactar". {"route":"contactar","route_parameters":{"_controller":"AppBundle\\Controller\\EmpresaController::contactarAction","marca":"a","_route":"contactar"},"request_uri":"http://127.0.0.1:8000/app.php/contactar/a","method":"GET"} []
[2017-08-04 12:01:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-08-04 12:01:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "Te has equivocado. No existe" at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php line 300 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): Te has equivocado. No existe at /var/www/html/app/app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php:300)"} []
